cmd/aro/rp.go

```diff
@@ -32,7 +32,9 @@ import (
 )
 
 func rp(ctx context.Context, log, audit *logrus.Entry) error {
-	_env, err := env.NewEnv(ctx, log)
+	stop := make(chan struct{})
+
+	_env, err := env.NewEnv(ctx, stop, log)
```

Comment:
The stop channel is essential for proper tests. Even when the function gets created always anew, the ticker leaks and breaks next iteration of tests.

Since the stop channel is required, I propagated it here to ensure proper cleanup.